cmd/ethereum/listener.go

```diff
@@ -66,7 +67,7 @@ func StartListener(cfg config.Config, logger log.Logger, processingQueue chan *t
 	for _, historicalLog := range history {
 		parsedMsg, err := types.EvmLogToMessageState(messageTransmitterABI, messageSent, &historicalLog)
 		if err != nil {
-			logger.Error("Unable to parse history log into MessageState, skipping")
+			logger.Error("Unable to parse history log into MessageState, skipping", "err", err)
```

Comment:
Nice. I had the same idea in order to help with issues like #9. 
I think this PR could close out that issue too.